packages/mobx-state-tree/src/core/node/object-node.ts

```diff
@@ -208,7 +208,7 @@ export class ObjectNode<C, S, T> extends BaseNode<C, S, T> {
 
         // initialize the uninitialized parent chain from parent to child
         for (const p of parentChain) {
-            p.createObservableInstanceIfNeeded()
+            p.createObservableInstanceIfNeeded(false)
```

Comment:
I don't think `...IfNeeded` is required here. We know these parents are not observable from the loop above. But I left it that way to minimize the number of changes. 

Perhaps there is a case where calling createObservableInstance on a parent triggers createObservableInstance on a child before this loop calls createObservableInstance itself.